docs/api/rest_api/rest_api_reference/input/ez-cart.raml

```diff
@@ -6,8 +6,8 @@ post:
         Content-Type:
             description: The CartCreateStruct schema encoded in XML or JSON format.
             example: |
-                application/vnd.ibexa.api.CartCreateStruct+xml
-                application/vnd.ibexa.api.CartCreateStruct+json
+                application/vnd.ibexa.api.CartCreate+xml
+                application/vnd.ibexa.api.CartCreate+json
     #body:
     #    application/vnd.ibexa.api.CartCreateStruct+xml:
```

Comment:
Could a dev provide a CartCreate usage example to get the body documented?

Reply:
https://github.com/ibexa/documentation-developer/pull/2172/commits/4c2a6188f7efee8eb7f7576019ce43e2081d8755